subsys/bluetooth/controller/ll_sw/ull_conn_types.h

```diff
@@ -470,13 +470,36 @@ struct llcp_struct {
 	uint8_t tx_q_pause_data_mask;
 }; /* struct llcp_struct */
 
+#if defined(CONFIG_BT_CTLR_PERIPHERAL_ISO)
+struct llcp_cis_struct {
+	enum {
+		LLCP_CIS_STATE_REQ,
+		LLCP_CIS_STATE_RSP_WAIT,
+		LLCP_CIS_STATE_IND_WAIT,
+		LLCP_CIS_STATE_INST_WAIT
+	} state:8 __packed;
+	uint8_t  cig_id;
+	uint16_t cis_handle;
+	uint8_t  cis_id;
+	uint32_t c_max_sdu:12;
```

Comment:
Could `uint16_t conn_event_count;` be placed between `cis_id` and `c_max_sdu`? 

At least to me it looks like that would be better in terms of grouping by word size of 32-bit.

Reply:
Yes, however this is the way it was defined for legacy.